tools/releaseBuild/signing.xml

```diff
@@ -6,7 +6,7 @@
          dest="__OUTPATHROOT__\release\out\Microsoft.PowerShell.Standard.Module.Template.0.1.4.nupkg" />
     <file  src="__INPATHROOT__\release\out\PowerShellStandard.Library.3.0.0-preview-02.nupkg" signType="CP-401405"
          dest="__OUTPATHROOT__\release\out\PowerShellStandard.Library.3.0.0-preview-02.nupkg" />
-    <file  src="__INPATHROOT__\release\out\PowerShellStandard.Library.5.1.1.nupkg" signType="CP-401405"
-         dest="__OUTPATHROOT__\release\out\PowerShellStandard.Library.5.1.1.nupkg" />
+    <file  src="__INPATHROOT__\release\out\PowerShellStandard.Library.5.1.0.nupkg" signType="CP-401405"
+         dest="__OUTPATHROOT__\release\out\PowerShellStandard.Library.5.1.0.nupkg" />
```

Comment:
@JamesWTruher can you speak to this change? Why does the signing.xml have 5.1.1?

Reply:
@JamesWTruher @TylerLeonhardt Any chance of getting this PR moved through?  It would be nice if the template folks installed with dotnet worked better.  :-)

Reply:
yah - sorry, I thought I had pushed 5.1.1 months ago. I'll make this happen